viYarn/Episode/model/Episode.swift

```diff
@@ -11,7 +11,24 @@ import Foundation
 
 struct Episode {
     
-    let title: String
-    let description: String
+    let isParsedSuccessfully: Bool
+    let posts: [EpisodePost]
+}
+extension Episode {
+    
+    init?(json: [String: Any]) {
+        guard let postsJson = json["posts"] as? [[String : Any]] else { return nil }
+        let successText = (json["status"] as? String) ?? "false"
+        let isParsedSuccessfully = successText == "ok" ? true : false
```

Comment:
p0:

let isParsedSuccessfully = successText == "ok" ? true : false 
to
let isParsedSuccessfully = successText == "ok"